lib/devil-bot-rust-cdk-stack.ts

```diff
@@ -20,7 +20,7 @@ export class DevilBotRustCdkStack extends cdk.Stack {
       environment: { // Fill in your personal app's webhook URLs below when testing (remove them when creating a PR)
         RUST_BACKTRACE: "1",
         SLACK_API_TOKEN: "",
-        DEVIL_BOT_TEST_CHANNEL_URL: "",
+        DEVIL_BOT_TEST_CHANNEL_URL: "https://hooks.slack.com/services/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",
```

Comment:
Don't push slack hook URLs. They will get scanned by Slack and deleted. Probably this already happened and you're going to get an email.